example/main.tf

```diff
@@ -0,0 +1,51 @@
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  namespace = "example"
+  stage     = "dev"
+  name      = "app"
+}
+
+module "subnets" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=master"
+  namespace          = "example"
+  stage              = "dev"
+  name               = "app"
+  region             = "us-west-2"
+  vpc_id             = "${module.vpc.vpc_id}"
+  igw_id             = "${module.vpc.igw_id}"
+  cidr_block         = "10.0.0.0/16"
+  availability_zones = ["us-west-2a", "us-west-2b"]
+}
+
+module "elasticsearch" {
+  source                         = "git::https://github.com/cloudposse/terraform-aws-elasticsearch.git?ref=tags/0.1.5"
+  namespace                      = "example"
```

Comment:
We don't want to hardcode these in examples b/c it precludes us testing the module concurrently. Additionally, sometimes our community will deploy this version without changing the values, which means we cannot reuse them.